### PR TITLE
Name change

### PR DESCRIPTION
**Pull request by gp1702**
Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [x] Reference the issue this is meant to fix or describe it if none exists.
- [x] Full description of changes made.

## Pull Request Checklist

- [x] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [x] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [x] The Docker image builds
- [x] The container runs correctly when sent inputs with the changes
- [x] No models or other large files are part of these commits
- [x] A description of the tests already run as part of this PR is present
- [ ] CI is set up under `.github/workflows` or is not applicable

### CODE REVIEW

**Comment by jeffbl** [comment]
@gp1702 assigned to Juliette on assumption this is ready for review, please self-assign if you still have pending work on this.

**Comment by JRegimbal** [comment]
Assigning back for more details on testing.

**Comment by gp1702** [comment]
Testing on Unicorn is now complete. The code works as expected.

File: .github/workflows/second-categoriser.yml [hunk 1]
```diff
@@ -63,11 +63,11 @@ jobs:
           flavor: |
             latest=${{ env.TAGGED }}
           tags: |
-            type=match,enable=${{ env.TAGGED }},priority=300,pattern=preprocessor-second-categoriser-(\d+.\d+.\d+),group=1
+            type=match,enable=${{ env.TAGGED }},priority=300,pattern=preprocessor-grphic-tagger-(\d+.\d+.\d+),group=1
```
Comment (JRegimbal) [inline .github/workflows/second-categoriser.yml:66]:
```suggestion
            type=match,enable=${{ env.TAGGED }},priority=300,pattern=preprocessor-graphic-tagger-(\d+.\d+.\d+),group=1
```

File: preprocessors/content-categoriser/categoriser.py [hunk 2]
```diff
@@ -83,7 +85,7 @@ def categorise():
         return "", 204  # No content
     request_uuid = content["request_uuid"]
     timestamp = time.time()
-    name = "ca.mcgill.a11y.image.preprocessor.firstCategoriser"
+    name = "ca.mcgill.a11y.image.preprocessor.content-categoriser"
```
Comment (JRegimbal) [inline preprocessors/content-categoriser/categoriser.py:88]: We typically use camel case for these.
```suggestion
    name = "ca.mcgill.a11y.image.preprocessor.contentCategoriser"
```

File: preprocessors/graphic-tagger/azure_api.py [hunk 3]
```diff
@@ -126,9 +128,9 @@ def categorise():
         return jsonify("Invalid Preprocessor JSON format"), 400
     request_uuid = content["request_uuid"]
     timestamp = time.time()
-    name = "ca.mcgill.a11y.image.preprocessor.secondCategoriser"
+    name = "ca.mcgill.a11y.image.preprocessor.graphic-tagger"
```
Comment (JRegimbal) [inline preprocessors/graphic-tagger/azure_api.py:131]:
```suggestion
    name = "ca.mcgill.a11y.image.preprocessor.graphicTagger"
```

File: preprocessors/graphic-tagger/azure_api.py [hunk 4]
```diff
@@ -126,9 +128,9 @@ def categorise():
...
     preprocess_output = content["preprocessors"]
-    classifier_1 = "ca.mcgill.a11y.image.preprocessor.firstCategoriser"
+    classifier_1 = "ca.mcgill.a11y.image.preprocessor.content-categoriser"
```
Comment (JRegimbal) [inline preprocessors/graphic-tagger/azure_api.py:133]:
```suggestion
    classifier_1 = "ca.mcgill.a11y.image.preprocessor.contentCategoriser"
```

File: preprocessors/semanticSeg/segment.py [hunk 5]
```diff
@@ -235,8 +235,8 @@ def segment():
     timestamp = time.time()
     preprocessorName = \
         "ca.mcgill.a11y.image.preprocessor.semanticSegmentation"
-    classifier_1 = "ca.mcgill.a11y.image.preprocessor.firstCategoriser"
-    classifier_2 = "ca.mcgill.a11y.image.preprocessor.secondCategoriser"
+    classifier_1 = "ca.mcgill.a11y.image.preprocessor.content-categoriser"
```
Comment (JRegimbal) [inline preprocessors/semanticSeg/segment.py:238]:
```suggestion
    classifier_1 = "ca.mcgill.a11y.image.preprocessor.contentCategoriser"
```

File: preprocessors/semanticSeg/segment.py [hunk 6]
```diff
@@ -235,8 +235,8 @@ def segment():
...
+    classifier_2 = "ca.mcgill.a11y.image.preprocessor.graphic-tagger"
```
Comment (JRegimbal) [inline preprocessors/semanticSeg/segment.py:239]:
```suggestion
    classifier_2 = "ca.mcgill.a11y.image.preprocessor.graphicTagger"
```

File: preprocessors/yolo/detect.py [hunk 7]
```diff
@@ -207,8 +207,8 @@ def run(weights='yolov5x.pt',
         image = np.asarray(bytearray(binary), dtype="uint8")
         imgDim = cv2.imdecode(image, cv2.IMREAD_COLOR)
         height, width, channels = imgDim.shape
-        classifier_1 = "ca.mcgill.a11y.image.preprocessor.firstCategoriser"
-        classifier_2 = "ca.mcgill.a11y.image.preprocessor.secondCategoriser"
+        classifier_1 = "ca.mcgill.a11y.image.preprocessor.content-categoriser"
```
Comment (JRegimbal) [inline preprocessors/yolo/detect.py:210]:
```suggestion
        classifier_1 = "ca.mcgill.a11y.image.preprocessor.contentCategoriser"
```

File: preprocessors/yolo/detect.py [hunk 8]
```diff
@@ -207,8 +207,8 @@ def run(weights='yolov5x.pt',
...
+        classifier_2 = "ca.mcgill.a11y.image.preprocessor.graphic-tagger"
```
Comment (JRegimbal) [inline preprocessors/yolo/detect.py:211]:
```suggestion
        classifier_2 = "ca.mcgill.a11y.image.preprocessor.graphicTagger"
```

**Review by JRegimbal — CHANGES_REQUESTED** [review]
Typo and switch to camel case.

**Comment by JRegimbal** [comment]
Tested on Unicorn with example images 2 and 10. All preprocessors run correctly without erroring and the photo audio handler runs correctly. Note that the other handlers impacted are either currently unused or in progress (photo-audio-haptics-handler).

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**